doc/index.rst

```diff
@@ -9,31 +9,94 @@ Welcome to Dune's Documentation!
 .. toctree::
    :maxdepth: 3
 
+   .. We include the titles of the pages here to make sure they are in
+      alphabetical order. Eventually we should name the files and titles
+      similarly.
+
+   .. This topics should appear first since they are the most important ones.
```

Comment:
```suggestion
   .. These topics should appear first since they are the most important ones.
```

Reply:
I had to get rid of the comments since they caused a warning.